modules/integration/src/test/scala/scala/cli/integration/BspTestDefinitions.scala

```diff
@@ -125,6 +125,38 @@ abstract class BspTestDefinitions(val scalaVersionOpt: Option[String])
     }
   }
 
+  test("setup-ide should succeed for valid dependencies") {
+    val inputs = TestInputs(
+      Seq(os.rel / "simple.sc" -> "import $ivy.`com.lihaoyi::pprint:0.6.6`")
+    )
+    inputs.fromRoot { root =>
+      val p = os.proc(
+        TestUtil.cli,
+        "setup-ide",
+        extraOptions,
+        "--dependency",
+        "com.lihaoyi::cask:0.7.11"
+      ).call(cwd = root, check = false)
+      expect(p.exitCode == 1)
```

Comment:
Isn't it supposed to succeed here?
```suggestion
      os.proc(
        TestUtil.cli,
        "setup-ide",
        extraOptions,
        "--dependency",
        "com.lihaoyi::cask:0.7.11"
      ).call(cwd = root)
```

Reply:
Thanks for finding this, the tests were failing, because in both cases the setup-cli command was failing earlier ("No inputs found"). Now I think it's correct. I also had to adapt to `CrossSources` returning `Either` after rebase.